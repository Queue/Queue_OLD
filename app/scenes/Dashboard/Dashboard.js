//
// Dashboard scene

import React, { Component } from 'react';
import { StyleSheet, View, Text } from 'react-native';
import { Grid, Col } from 'react-native-easy-grid';

export default class Dashboard extends Component {
  constructor(props) {
    super(props);

    this.state = {};
  }

  render() {
    return (
      <Grid>

        <Col style={{backgroundColor: 'red', width: 110}}></Col>

        <Col style={{backgroundColor: '#eaeaea'}}></Col>

        <Col style={{backgroundColor: 'blue', width: 350}}></Col>

      </Grid>
    );
  }
};

const styles = StyleSheet.create({
  default: {
    backgroundColor: 'white'
  }
});
