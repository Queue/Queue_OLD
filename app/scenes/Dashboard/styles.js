//
// Dashboard styles

import { StyleSheet } from 'react-native';

export default styles = StyleSheet.create({
  default: {
    backgroundColor: 'white'
  },
  logOutButton: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    padding: 20
  },
  navMenu: {
    borderRightWidth: 1,
    borderColor: Colors.primaryBackground,
    maxWidth: 110
  },
  actionArea: {
    borderRightWidth: 1,
    borderColor: Colors.primaryBackground,
    minWidth: 525
  },
  queueList: {
    backgroundColor: 'transparent'
  },
  addButton: {
    alignItems: 'center',
    backgroundColor: Colors.primaryForeground,
    maxHeight: 60
  },
  addButtonText: {
    color: 'white',
    fontSize: 55,
    marginTop: -5
  }
});
