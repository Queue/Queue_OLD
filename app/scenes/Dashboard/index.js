//
// Dashboard index

import Dashboard from './Dashboard';
export default Dashboard;
