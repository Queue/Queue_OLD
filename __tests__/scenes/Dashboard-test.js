//
// Dashboard scene test

import React from 'react';
import Dashboard from '../../app/scenes/Dashboard';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Dashboard />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
