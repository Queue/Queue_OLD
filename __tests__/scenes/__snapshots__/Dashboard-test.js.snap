exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
    }
  }>
  <View
    style={
      Object {
        "backgroundColor": "red",
        "flex": 0,
        "flexDirection": "column",
        "width": 110,
      }
    } />
  <View
    style={
      Object {
        "backgroundColor": "#eaeaea",
        "flex": 1,
        "flexDirection": "column",
      }
    } />
  <View
    style={
      Object {
        "backgroundColor": "blue",
        "flex": 0,
        "flexDirection": "column",
        "width": 350,
      }
    } />
</View>
`;
